docs/spark/how-to-guides/connect-to-sql-server.md

```diff
@@ -0,0 +1,68 @@
+---
+title: Connect to SQL server locally from .NET for Apache Spark
+description: Learn how to connect to an SQL server instance from your .NET for Apache Spark application.
+ms.date: 09/24/2020
+ms.topic: conceptual
+ms.custom: mvc,how-to
+---
+
+# Connect to SQL server locally from .NET for Apache Spark
+
+In this article, you learn how to connect to an SQL server instance from your [.NET for Apache Spark](https://github.com/dotnet/spark) application.
+
+## Configure your SQL server to allow access from your .NET for Apache Spark application
+
+1. Add a Login user and password choosing SQL server authentication to your SQL server instance.
+2. Give that login user necessary permissions at the relevant database level as shown below:
+
+    ![SQL server permissions](./media/connect-external-sources/SqlServerAuth.png)
+
+3. Make sure the default port for SQL server 1433 is allowed through the firewall.
+4. Open SQL configure manager to enable TCP/IP through the network configuration as shown below:
+
+    ![SQL server TCP/IP enable](./media/connect-external-sources/SqlServerTCPIP.png)
+
+    Also note the value of `Listen All` in above tab under `Protocol`.
+
+5. Configure the TCP/IP port to 1433 for all required IP addresses if the `Listen All` is set to `No`. Otherwise, set the TCP Port in IPAll.
```

Comment:
```suggestion
5. Configure the TCP/IP port to 1433 for all required IP addresses if the **Listen All** is set to **No**. Otherwise, set the TCP Port in IPAll.
```

Reply:
@Niharikadutta since these are UI elements and settings to look out for, it makes sense to highlight them